packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2741,7 +2743,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             // Please note that this does not change file format, so it can be disabled in the future if this
             // optimization no longer makes sense (for example, batch compression may make it less appealing).
             if (this.currentlyBatching() && type === ContainerMessageType.Attach &&
-                this.mc.config.getBoolean("Fluid.ContainerRuntime.enableAttachOpReorder") === true) {
+                this.mc.config.getBoolean("Fluid.ContainerRuntime.disableAttachOpReorder") !== true) {
```

Comment:
This was disabled for a particular reason which I can't remember now. Is this current enabled via the enableAttachOpReorder feature flag? Are we confident we can enable it by default?

Reply:
I think it was disabled because of the test failing during the merge.

Reply:
Yep, I told Mark to not waste time on it, and then followed up https://github.com/microsoft/FluidFramework/pull/12103 to fix a bug discovered and re-enable it